android/titanium/src/java/org/appcelerator/titanium/TiC.java

```diff
@@ -1424,7 +1424,12 @@
 	 * @module.api
 	 */
 	public static final String PROPERTY_FONT_SIZE = "font-size";
-
+	
+	/**
+	 * @module.api
+	 */
+	public static final String PROPERTY_MINIMUM_FONT_SIZE = "minimumFontSize";
```

Comment:
It has to be `PROPERTY_MINIMUM_FONT_SIZE`. The constants are based on the camel case name that injects `_` chars instead of the camel-case style.